Classes/User.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+
+
+class User:
+	def __init__(self):
+		self.UserName = ''
+		self.Password = ''
+		self.FirstName = ''
+		self.LastName = ''
+	
+	
+	def getUserName(self):
+		return self.UserName
+		
+	def setUserName(self, name):
+		self.UserName = name
+	
+	
+	def getPassword(self):
+		return self.Password
+		
+	def setPassword(self, password):
+		self.Password = password
+	
+	
+	def getFirstName(self):
+		return self.FirstName
+		
+	def setFirstName(self, name):
+		self.FirstName = name
+	
+	
+	def getLastName(self):
+		return self.LastName
+		
+	def setLastName(self, name):
+		self.LastName = name
+		
+		
+	def writeUserFileName(self):
+		filename = '/home/iceman/_/code/chay-family-footsteps/Data/User/' + self.UserName + '.json'
+		return filename
+	
+	def __str__(self):
+		return '''
+Username: {}
+Password: {}
+First Name: {}
+Last Name: {}
+''' .format(self.UserName, self.Password, self.FirstName, self.LastName)#Login().getUserName(), Login().getPassword(), Login().getFirstName(), Login().getLastName())
+		
+	def jsonDefault(self, object):
+		return object.__dict__
```

Comment:
Hey,C2,This proggram dos'nt work!

(~~~~
(~~~~  /

Reply:
I'll fix it. There is a name error. I'll push up the fixed version.